src/client/settings.tsx

```diff
@@ -13,6 +13,7 @@ export type RRSettings = ReadonlyDeep<{
   renderMode: "fast" | "mostly-fancy" | "fancy";
   enableExperimental25D: boolean;
   collapseDiceTemplates: boolean;
+  focusTokenOnTurnStart: boolean;
```

Comment:
Can you increment `CURRENT_SETTINGS_VERSION`? Otherwise `focusTokenOnTurnStart` will be `undefined` for people until they toggle it at least once.